nautobot_circuit_maintenance/app-config-schema.json

```diff
@@ -0,0 +1 @@
+true
```

Comment:
@snaselj could you elaborate about this?

Reply:
Going to be resolved here:
https://github.com/nautobot/nautobot-app-circuit-maintenance/pull/286